lib/analyzer/static-analyzer.ts

```diff
@@ -87,7 +87,7 @@ export async function analyze(
     );
   }
 
-  const appScan = isTrue(options["app-vulns"]);
+  const appScan = !isTrue(options["exclude-app-vulns"]);
```

Comment:
I'm not sure I understand the question. the syntax for the CLI will be `snyk container test --exclude-app-vulns <image_name>` with no need to specify `=true`. It's passed to the plugin as `true` from the CLI.
We only have `isTrue`, which checks that an option exists and equals `true`. Anything else is false. Note that a function `isFalse` that will do the opposite (exists and equals `false`) will not work for this flag, because we consider it `false` if the option is missing entirely from the options object.

Reply:
Ohh ok I understand now, thanks for the detailed explanation!